src/web/src/config.ecpp

```diff
@@ -124,7 +124,7 @@ UserInfo user;
         {
             std::string key = qparam.param ("key");
             if (!key_format.match (key)) {
-                std::string msg = "satisfy format " + key_format_string;
+                std::string msg = TRANSLATE_ME ("satisfy format " + key_format_string);
```

Comment:
This would need to be updated in the end, as getting the strings step (simple grep TRANSLATE_ME) would get either key {"satisfy format "} or {"satisfy format " + key_format_string}, but neither will be valid in runtime, as it will be evaluation of the string, so runtime key wouldn't match grepped key.